tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -730,6 +730,26 @@ def test_docstring(a: cases.IField):
     cases.verify(cartesian_case, test_docstring, a, inout=a, ref=a)
 
 
+def test_with_bound_args(cartesian_case):
+    @gtx.field_operator
+    def fieldop_bound_args(a: cases.IField, scalar: int32, bool_val: bool) -> cases.IField:
+        if not bool_val:
+            scalar = int32(0)
+        return a + a + scalar
```

Comment:
```suggestion
    def fieldop_bound_args(a: cases.IField, scalar: int32, condition: bool) -> cases.IField:
        if not condition:
            scalar = 0
        return a + scalar
```
Let's also parameterize the test on the condition.

Reply:
I will change bool_val and scalar but not the return statement